pkg/domain/infra/abi/containers.go

```diff
@@ -1319,6 +1319,15 @@ func (ic *ContainerEngine) ContainerStats(ctx context.Context, namesOrIds []stri
 	if options.Interval < 1 {
 		return nil, errors.New("Invalid interval, must be a positive number greater zero")
 	}
+	if rootless.IsRootless() {
```

Comment:
Just to verify, the API calls this function as well, so we don't need the change there?